### ISSUE TITLE
webrtc_PeerCC to PeerCC-browser 

### ISSUE THREAD

**Issue by Muhayyuddin** [OpticalTone/PeerCC-Browser#10]
Hi, I would like to video call from UWP PeerCC app to Browser or from HL peerCC (unity) to browser.
with PeerCC-browser example I am able to call successfully from web-browser to web-browser. but when I try to call from UWP peerCC sample, I am not able to see the remote stream at browser.
browser --> browser : works fine
browser --> uwp peerCC : at peercc i can see local and remote video but on browser side it just show local video.
HL --> browser same result.
it also didnot show any error.
any potential solution or suggestion to figure it out (why remote video is not appear at browser) will be really appreciated.
thank you.

**Comment by Muhayyuddin** [comment]
Hi, I would like to call from UWP PeerCC app to Browser or from HL peerCC to browser. 
with PeerCC-browser example I am able to call successfully from web-browser to web-browser. but when I try to call from UWP peerCC sample, I am not able to see the remote stream at browser.
browser --> browser : works fine
browser --> uwp peerCC : at peercc i can see local and remote video but on browser it just show local video. 
HL --> browser same result. 
it also didnot show any error. 
any potential solution or suggestion to figure it out (why remote video is not appear at browser) will be really appreciated. 
thank you.

**Comment by Muhayyuddin** [comment]
For uwp PeerCC i am using the branch m71 (unity fix) i compiled it from source.

**Comment by Muhayyuddin** [comment]
here is the console output when i call from browser to peercc.
```script
WebRTC is supported.
signallingchannel.js:356 Info: {"address":"IP removed for security reason","port":"8888"}
signallingchannel.js:378 Server:http://IP removed for security reason:8888
signallingchannel.js:32 pragma: 1
content-type: text/plain
cache-control: no-cache
content-length: 18

signallingchannel.js:385 start responce: MyBrowserPeer,1,1

signallingchannel.js:390 Array(2)
RTC.js:1430 Connected to Peer Connection Server
RTC.js:1442 {"registerdone":{"id":"1","friendlyName":"MyBrowserPeer"}}
RTC.js:1479 {"id":"1","friendlyName":"MyBrowserPeer"}
RTC.js:1430 Connected to Peer Connection Server
signallingchannel.js:73 pragma: 1
content-type: text/plain
cache-control: no-cache
content-length: 20

signallingchannel.js:312 UWPPeer,2,1

RTC.js:1442 {"contacts":"UWPPeer,2,1\n"}
RTC.js:278 Object
RTC.js:279 Media constraints changed.
RTC.js:650 Using device: Default - Microphone Array (Realtek High Definition Audio)
RTC.js:650 Using device: B525 HD Webcam (046d:0836)
signallingchannel.js:185 Sdp Sent: {"type":"offer","sdp":"v=0\r\no=- 2144224120787931323 2 IN IP4 127.0.0.1\r\ns=-\r\nt=0 0\r\na=group:BUNDLE 0 1\r\na=msid-semantic: WMS g5YlGczyYsLRmycXUHAPqACOnB4ZBKlLZw0u\r\nm=audio 9 UDP/TLS/RTP/SAVPF 111 103 104 9 0 8 106 105 13 110 112 113 126\r\nc=IN IP4 0.0.0.0\r\na=rtcp:9 IN IP4 0.0.0.0\r\na=ice-ufrag:/N+H\r\na=ice-pwd:eaUG/0fnxUK9mL0Xp7Mqra3K\r\na=ice-options:trickle\r\na=fingerprint:sha-256 52:8C:A9:64:E6:E3:4F:A3:04:8F:F2:55:4C:9B:78:56:A5:9C:19:C6:0B:D0:F5:AC:5F:16:4D:A5:E5:37:68:1B\r\na=setup:actpass\r\na=mid:0\r\na=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level\r\na=extmap:2 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01\r\na=extmap:3 urn:ietf:params:rtp-hdrext:sdes:mid\r\na=extmap:4 urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id\r\na=extmap:5 urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id\r\na=sendrecv\r\na=msid:g5YlGczyYsLRmycXUHAPqACOnB4ZBKlLZw0u 2a964532-4538-46f2-a6d4-0048561067ec\r\na=rtcp-mux\r\na=rtpmap:111 opus/48000/2\r\na=rtcp-fb:111 transport-cc\r\na=fmtp:111 minptime=10;useinbandfec=1\r\na=rtpmap:103 ISAC/16000\r\na=rtpmap:104 ISAC/32000\r\na=rtpmap:9 G722/8000\r\na=rtpmap:0 PCMU/8000\r\na=rtpmap:8 PCMA/8000\r\na=rtpmap:106 CN/32000\r\na=rtpmap:105 CN/16000\r\na=rtpmap:13 CN/8000\r\na=rtpmap:110 telephone-event/48000\r\na=rtpmap:112 telephone-event/32000\r\na=rtpmap:113 telephone-event/16000\r\na=rtpmap:126 telephone-event/8000\r\na=ssrc:531314222 cname:ilWXt/pw5F9hXhI1\r\na=ssrc:531314222 msid:g5YlGczyYsLRmycXUHAPqACOnB4ZBKlLZw0u 2a964532-4538-46f2-a6d4-0048561067ec\r\na=ssrc:531314222 mslabel:g5YlGczyYsLRmycXUHAPqACOnB4ZBKlLZw0u\r\na=ssrc:531314222 label:2a964532-4538-46f2-a6d4-0048561067ec\r\nm=video 9 UDP/TLS/RTP/SAVPF 96 97 98 99 100 101 102 122 127 121 125 107 108 109 124 120 123 119 114 115 116\r\nc=IN IP4 0.0.0.0\r\na=rtcp:9 IN IP4 0.0.0.0\r\na=ice-ufrag:/N+H\r\na=ice-pwd:eaUG/0fnxUK9mL0Xp7Mqra3K\r\na=ice-options:trickle\r\na=fingerprint:sha-256 52:8C:A9:64:E6:E3:4F:A3:04:8F:F2:55:4C:9B:78:56:A5:9C:19:C6:0B:D0:F5:AC:5F:16:4D:A5:E5:37:68:1B\r\na=setup:actpass\r\na=mid:1\r\na=extmap:14 urn:ietf:params:rtp-hdrext:toffset\r\na=extmap:13 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time\r\na=extmap:12 urn:3gpp:video-orientation\r\na=extmap:2 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01\r\na=extmap:11 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay\r\na=extmap:6 http://www.webrtc.org/experiments/rtp-hdrext/video-content-type\r\na=extmap:7 http://www.webrtc.org/experiments/rtp-hdrext/video-timing\r\na=extmap:8 http://tools.ietf.org/html/draft-ietf-avtext-framemarking-07\r\na=extmap:9 http://www.webrtc.org/experiments/rtp-hdrext/color-space\r\na=extmap:3 urn:ietf:params:rtp-hdrext:sdes:mid\r\na=extmap:4 urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id\r\na=extmap:5 urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id\r\na=sendrecv\r\na=msid:g5YlGczyYsLRmycXUHAPqACOnB4ZBKlLZw0u 933b5d38-8834-4443-bbe2-0b6b54ba7289\r\na=rtcp-mux\r\na=rtcp-rsize\r\na=rtpmap:96 VP8/90000\r\na=rtcp-fb:96 goog-remb\r\na=rtcp-fb:96 transport-cc\r\na=rtcp-fb:96 ccm fir\r\na=rtcp-fb:96 nack\r\na=rtcp-fb:96 nack pli\r\na=rtpmap:97 rtx/90000\r\na=fmtp:97 apt=96\r\na=rtpmap:98 VP9/90000\r\na=rtcp-fb:98 goog-remb\r\na=rtcp-fb:98 transport-cc\r\na=rtcp-fb:98 ccm fir\r\na=rtcp-fb:98 nack\r\na=rtcp-fb:98 nack pli\r\na=fmtp:98 profile-id=0\r\na=rtpmap:99 rtx/90000\r\na=fmtp:99 apt=98\r\na=rtpmap:100 VP9/90000\r\na=rtcp-fb:100 goog-remb\r\na=rtcp-fb:100 transport-cc\r\na=rtcp-fb:100 ccm fir\r\na=rtcp-fb:100 nack\r\na=rtcp-fb:100 nack pli\r\na=fmtp:100 profile-id=2\r\na=rtpmap:101 rtx/90000\r\na=fmtp:101 apt=100\r\na=rtpmap:102 H264/90000\r\na=rtcp-fb:102 goog-remb\r\na=rtcp-fb:102 transport-cc\r\na=rtcp-fb:102 ccm fir\r\na=rtcp-fb:102 nack\r\na=rtcp-fb:102 nack pli\r\na=fmtp:102 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42001f\r\na=rtpmap:122 rtx/90000\r\na=fmtp:122 apt=102\r\na=rtpmap:127 H264/90000\r\na=rtcp-fb:127 goog-remb\r\na=rtcp-fb:127 transport-cc\r\na=rtcp-fb:127 ccm fir\r\na=rtcp-fb:127 nack\r\na=rtcp-fb:127 nack pli\r\na=fmtp:127 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42001f\r\na=rtpmap:121 rtx/90000\r\na=fmtp:121 apt=127\r\na=rtpmap:125 H264/90000\r\na=rtcp-fb:125 goog-remb\r\na=rtcp-fb:125 transport-cc\r\na=rtcp-fb:125 ccm fir\r\na=rtcp-fb:125 nack\r\na=rtcp-fb:125 nack pli\r\na=fmtp:125 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f\r\na=rtpmap:107 rtx/90000\r\na=fmtp:107 apt=125\r\na=rtpmap:108 H264/90000\r\na=rtcp-fb:108 goog-remb\r\na=rtcp-fb:108 transport-cc\r\na=rtcp-fb:108 ccm fir\r\na=rtcp-fb:108 nack\r\na=rtcp-fb:108 nack pli\r\na=fmtp:108 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42e01f\r\na=rtpmap:109 rtx/90000\r\na=fmtp:109 apt=108\r\na=rtpmap:124 H264/90000\r\na=rtcp-fb:124 goog-remb\r\na=rtcp-fb:124 transport-cc\r\na=rtcp-fb:124 ccm fir\r\na=rtcp-fb:124 nack\r\na=rtcp-fb:124 nack pli\r\na=fmtp:124 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=4d0032\r\na=rtpmap:120 rtx/90000\r\na=fmtp:120 apt=124\r\na=rtpmap:123 H264/90000\r\na=rtcp-fb:123 goog-remb\r\na=rtcp-fb:123 transport-cc\r\na=rtcp-fb:123 ccm fir\r\na=rtcp-fb:123 nack\r\na=rtcp-fb:123 nack pli\r\na=fmtp:123 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=640032\r\na=rtpmap:119 rtx/90000\r\na=fmtp:119 apt=123\r\na=rtpmap:114 red/90000\r\na=rtpmap:115 rtx/90000\r\na=fmtp:115 apt=114\r\na=rtpmap:116 ulpfec/90000\r\na=ssrc-group:FID 697348605 4217768001\r\na=ssrc:697348605 cname:ilWXt/pw5F9hXhI1\r\na=ssrc:697348605 msid:g5YlGczyYsLRmycXUHAPqACOnB4ZBKlLZw0u 933b5d38-8834-4443-bbe2-0b6b54ba7289\r\na=ssrc:697348605 mslabel:g5YlGczyYsLRmycXUHAPqACOnB4ZBKlLZw0u\r\na=ssrc:697348605 label:933b5d38-8834-4443-bbe2-0b6b54ba7289\r\na=ssrc:4217768001 cname:ilWXt/pw5F9hXhI1\r\na=ssrc:4217768001 msid:g5YlGczyYsLRmycXUHAPqACOnB4ZBKlLZw0u 933b5d38-8834-4443-bbe2-0b6b54ba7289\r\na=ssrc:4217768001 mslabel:g5YlGczyYsLRmycXUHAPqACOnB4ZBKlLZw0u\r\na=ssrc:4217768001 label:933b5d38-8834-4443-bbe2-0b6b54ba7289\r\n"}
2signallingchannel.js:205 SEND SDP candidate
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:198 
signallingchannel.js:312 
2signallingchannel.js:205 SEND SDP candidate
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
6signallingchannel.js:205 SEND SDP candidate
signallingchannel.js:73 pragma: 2
content-type: text/plain
cache-control: no-cache
content-length: 149

signallingchannel.js:312 {"sdpMid":"0","sdpMLineIndex":0,"candidate":"candidate:2999745851 1 udp 2122260223 192.168.56.1 59768 typ host generation 0 ufrag KtBp network-id 1"}
signallingchannel.js:511 RECV: direct candidate
RTC.js:1442 {"DirectCandidate":{"sdpMid":"0","sdpMLineIndex":0,"candidate":"candidate:2999745851 1 udp 2122260223 192.168.56.1 59768 typ host generation 0 ufrag KtBp network-id 1"}}
RTC.js:1575 Ice candidate postponed!
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:205 SEND SDP candidate
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
signallingchannel.js:205 SEND SDP candidate
signallingchannel.js:73 pragma: 2
content-type: text/plain
cache-control: no-cache
content-length: 3085

signallingchannel.js:312 {"type":"answer","sdp":"v=0\r\no=- 9003934833620117014 2 IN IP4 127.0.0.1\r\ns=-\r\nt=0 0\r\na=group:BUNDLE 0 1\r\na=msid-semantic: WMS\r\nm=audio 9 UDP/TLS/RTP/SAVPF 111 103 104 9 0 8 106 105 13 110 112 113 126\r\nc=IN IP4 0.0.0.0\r\na=rtcp:9 IN IP4 0.0.0.0\r\na=ice-ufrag:KtBp\r\na=ice-pwd:trjB6tpXj52O/AY5+8QHPoLG\r\na=ice-options:trickle\r\na=fingerprint:sha-256 92:D8:3F:C2:5F:3B:CC:EA:C5:22:DF:72:F5:10:BE:9E:8A:E3:74:2E:82:B3:02:0D:D6:83:6B:62:EC:CC:59:76\r\na=setup:active\r\na=mid:0\r\na=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level\r\na=extmap:3 urn:ietf:params:rtp-hdrext:sdes:mid\r\na=sendrecv\r\na=msid:- SELF_AUDIO\r\na=rtcp-mux\r\na=rtpmap:111 opus/48000/2\r\na=rtcp-fb:111 transport-cc\r\na=fmtp:111 minptime=10;useinbandfec=1\r\na=rtpmap:103 ISAC/16000\r\na=rtpmap:104 ISAC/32000\r\na=rtpmap:9 G722/8000\r\na=rtpmap:0 PCMU/8000\r\na=rtpmap:8 PCMA/8000\r\na=rtpmap:106 CN/32000\r\na=rtpmap:105 CN/16000\r\na=rtpmap:13 CN/8000\r\na=rtpmap:110 telephone-event/48000\r\na=rtpmap:112 telephone-event/32000\r\na=rtpmap:113 telephone-event/16000\r\na=rtpmap:126 telephone-event/8000\r\na=ssrc:1765065967 cname:Ncm4kQcM8uN0NWmQ\r\nm=video 9 UDP/TLS/RTP/SAVPF 96 97 98 99 125 107 114 115 116\r\nc=IN IP4 0.0.0.0\r\na=rtcp:9 IN IP4 0.0.0.0\r\na=ice-ufrag:KtBp\r\na=ice-pwd:trjB6tpXj52O/AY5+8QHPoLG\r\na=ice-options:trickle\r\na=fingerprint:sha-256 92:D8:3F:C2:5F:3B:CC:EA:C5:22:DF:72:F5:10:BE:9E:8A:E3:74:2E:82:B3:02:0D:D6:83:6B:62:EC:CC:59:76\r\na=setup:active\r\na=mid:1\r\na=extmap:14 urn:ietf:params:rtp-hdrext:toffset\r\na=extmap:13 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time\r\na=extmap:12 urn:3gpp:video-orientation\r\na=extmap:2 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01\r\na=extmap:11 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay\r\na=extmap:6 http://www.webrtc.org/experiments/rtp-hdrext/video-content-type\r\na=extmap:7 http://www.webrtc.org/experiments/rtp-hdrext/video-timing\r\na=extmap:8 http://tools.ietf.org/html/draft-ietf-avtext-framemarking-07\r\na=extmap:3 urn:ietf:params:rtp-hdrext:sdes:mid\r\na=sendrecv\r\na=msid:- SELF_VIDEO\r\na=rtcp-mux\r\na=rtcp-rsize\r\na=rtpmap:96 VP8/90000\r\na=rtcp-fb:96 goog-remb\r\na=rtcp-fb:96 transport-cc\r\na=rtcp-fb:96 ccm fir\r\na=rtcp-fb:96 nack\r\na=rtcp-fb:96 nack pli\r\na=rtpmap:97 rtx/90000\r\na=fmtp:97 apt=96\r\na=rtpmap:98 VP9/90000\r\na=rtcp-fb:98 goog-remb\r\na=rtcp-fb:98 transport-cc\r\na=rtcp-fb:98 ccm fir\r\na=rtcp-fb:98 nack\r\na=rtcp-fb:98 nack pli\r\na=fmtp:98 x-google-profile-id=0\r\na=rtpmap:99 rtx/90000\r\na=fmtp:99 apt=98\r\na=rtpmap:125 H264/90000\r\na=rtcp-fb:125 goog-remb\r\na=rtcp-fb:125 transport-cc\r\na=rtcp-fb:125 ccm fir\r\na=rtcp-fb:125 nack\r\na=rtcp-fb:125 nack pli\r\na=fmtp:125 packetization-mode=1;profile-level-id=42e01f\r\na=rtpmap:107 rtx/90000\r\na=fmtp:107 apt=125\r\na=rtpmap:114 red/90000\r\na=rtpmap:115 rtx/90000\r\na=fmtp:115 apt=114\r\na=rtpmap:116 ulpfec/90000\r\na=ssrc-group:FID 2792671086 2571706030\r\na=ssrc:2792671086 cname:Ncm4kQcM8uN0NWmQ\r\na=ssrc:2571706030 cname:Ncm4kQcM8uN0NWmQ\r\n"}
signallingchannel.js:500 RECV: direct sdp
signallingchannel.js:73 pragma: 2
content-type: text/plain
cache-control: no-cache
content-length: 165

signallingchannel.js:312 {"sdpMid":"0","sdpMLineIndex":0,"candidate":"candidate:2072139682 1 udp 2122194687 192.168.1.59 59769 typ host generation 0 ufrag KtBp network-id 2 network-cost 10"}
signallingchannel.js:511 RECV: direct candidate
RTC.js:1442 {"SDP":{"peerInfo":{"id":"2"},"type":"answer","sdp":"v=0\r\no=- 9003934833620117014 2 IN IP4 127.0.0.1\r\ns=-\r\nt=0 0\r\na=group:BUNDLE 0 1\r\na=msid-semantic: WMS\r\nm=audio 9 UDP/TLS/RTP/SAVPF 111 103 104 9 0 8 106 105 13 110 112 113 126\r\nc=IN IP4 0.0.0.0\r\na=rtcp:9 IN IP4 0.0.0.0\r\na=ice-ufrag:KtBp\r\na=ice-pwd:trjB6tpXj52O/AY5+8QHPoLG\r\na=ice-options:trickle\r\na=fingerprint:sha-256 92:D8:3F:C2:5F:3B:CC:EA:C5:22:DF:72:F5:10:BE:9E:8A:E3:74:2E:82:B3:02:0D:D6:83:6B:62:EC:CC:59:76\r\na=setup:active\r\na=mid:0\r\na=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level\r\na=extmap:3 urn:ietf:params:rtp-hdrext:sdes:mid\r\na=sendrecv\r\na=msid:- SELF_AUDIO\r\na=rtcp-mux\r\na=rtpmap:111 opus/48000/2\r\na=rtcp-fb:111 transport-cc\r\na=fmtp:111 minptime=10;useinbandfec=1\r\na=rtpmap:103 ISAC/16000\r\na=rtpmap:104 ISAC/32000\r\na=rtpmap:9 G722/8000\r\na=rtpmap:0 PCMU/8000\r\na=rtpmap:8 PCMA/8000\r\na=rtpmap:106 CN/32000\r\na=rtpmap:105 CN/16000\r\na=rtpmap:13 CN/8000\r\na=rtpmap:110 telephone-event/48000\r\na=rtpmap:112 telephone-event/32000\r\na=rtpmap:113 telephone-event/16000\r\na=rtpmap:126 telephone-event/8000\r\na=ssrc:1765065967 cname:Ncm4kQcM8uN0NWmQ\r\nm=video 9 UDP/TLS/RTP/SAVPF 96 97 98 99 125 107 114 115 116\r\nc=IN IP4 0.0.0.0\r\na=rtcp:9 IN IP4 0.0.0.0\r\na=ice-ufrag:KtBp\r\na=ice-pwd:trjB6tpXj52O/AY5+8QHPoLG\r\na=ice-options:trickle\r\na=fingerprint:sha-256 92:D8:3F:C2:5F:3B:CC:EA:C5:22:DF:72:F5:10:BE:9E:8A:E3:74:2E:82:B3:02:0D:D6:83:6B:62:EC:CC:59:76\r\na=setup:active\r\na=mid:1\r\na=extmap:14 urn:ietf:params:rtp-hdrext:toffset\r\na=extmap:13 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time\r\na=extmap:12 urn:3gpp:video-orientation\r\na=extmap:2 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01\r\na=extmap:11 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay\r\na=extmap:6 http://www.webrtc.org/experiments/rtp-hdrext/video-content-type\r\na=extmap:7 http://www.webrtc.org/experiments/rtp-hdrext/video-timing\r\na=extmap:8 http://tools.ietf.org/html/draft-ietf-avtext-framemarking-07\r\na=extmap:3 urn:ietf:params:rtp-hdrext:sdes:mid\r\na=sendrecv\r\na=msid:- SELF_VIDEO\r\na=rtcp-mux\r\na=rtcp-rsize\r\na=rtpmap:96 VP8/90000\r\na=rtcp-fb:96 goog-remb\r\na=rtcp-fb:96 transport-cc\r\na=rtcp-fb:96 ccm fir\r\na=rtcp-fb:96 nack\r\na=rtcp-fb:96 nack pli\r\na=rtpmap:97 rtx/90000\r\na=fmtp:97 apt=96\r\na=rtpmap:98 VP9/90000\r\na=rtcp-fb:98 goog-remb\r\na=rtcp-fb:98 transport-cc\r\na=rtcp-fb:98 ccm fir\r\na=rtcp-fb:98 nack\r\na=rtcp-fb:98 nack pli\r\na=fmtp:98 x-google-profile-id=0\r\na=rtpmap:99 rtx/90000\r\na=fmtp:99 apt=98\r\na=rtpmap:125 H264/90000\r\na=rtcp-fb:125 goog-remb\r\na=rtcp-fb:125 transport-cc\r\na=rtcp-fb:125 ccm fir\r\na=rtcp-fb:125 nack\r\na=rtcp-fb:125 nack pli\r\na=fmtp:125 packetization-mode=1;profile-level-id=42e01f\r\na=rtpmap:107 rtx/90000\r\na=fmtp:107 apt=125\r\na=rtpmap:114 red/90000\r\na=rtpmap:115 rtx/90000\r\na=fmtp:115 apt=114\r\na=rtpmap:116 ulpfec/90000\r\na=ssrc-group:FID 2792671086 2571706030\r\na=ssrc:2792671086 cname:Ncm4kQcM8uN0NWmQ\r\na=ssrc:2571706030 cname:Ncm4kQcM8uN0NWmQ\r\n"}}
RTC.js:1442 {"DirectCandidate":{"sdpMid":"0","sdpMLineIndex":0,"candidate":"candidate:2072139682 1 udp 2122194687 192.168.1.59 59769 typ host generation 0 ufrag KtBp network-id 2 network-cost 10"}}
RTC.js:1570 Remote SDP candidate:
{"sdpMid":"0","sdpMLineIndex":0,"candidate":"candidate:2072139682 1 udp 2122194687 192.168.1.59 59769 typ host generation 0 ufrag KtBp network-id 2 network-cost 10"}
2RTC.js:617 Remote track resceved
RTC.js:1599 Postponed ice candidate added.
signallingchannel.js:46 content-type: text/plain
cache-control: no-cache
content-length: 0

signallingchannel.js:217 
signallingchannel.js:312 
```

**Comment by Muhayyuddin** [comment]
@nikolamand cc @jamescadd With some further investigation I found that 
the function ontrack called but event.stream is null. it is not receiving the remote stream. 
``` javascript
    //display remote track in the video element
    pc.ontrack = function(event) {
      videoRenderer = document.getElementById("rtcRenderer");
        videoRenderer.srcObject = event.streams[0];
      console.log("Remote track resceved");
      document.body.style.cursor = "default";
    };
    updateServerStatus();
  }
```
in your opinion what could be the reason? the issue of ICE? or could it be the issue of appropriate codec? I am trying with google chrome default audio video codec.

**Comment by nikolamand** [comment]
Hi, sorry for the late reply. I have encountered this issue before and I am not sure what exactly is causing it, I did some testing to figure out what was wrong, since this was a while back I don't remember all the details but I do remember I tried the following:
I was doing testing in the localhost so I don't think there is an issue with ice candidates, I also remember trying all possible combinations of the codecs, and some codec combinations worked in the way you have described(onlu UWP PeerCC sample showed remote video) other combinations didn't work at all. 
I can confirm that this issue happen only after m71 versions of the UWP, so the version m66 works the way it was intended. I am not sure if this helps you in any way, you can try to debug locally by running the peerconnection_server.exe, using the IP address as the server and setting the 'proxy' variable in signallingchannel.js to false, this should speed up the testing.

**Comment by Muhayyuddin** [comment]
@nikolamand Thank you for your reply. Yes I am testing with the local server I will updated if i found something interesting towards the debugging.

**Comment by Muhayyuddin** [comment]
@nikolamand  I think it the version compatibility issue it works fine the the previous version of google chrome. I am able to call from PeerCC UWP to browser.

**Comment by jamescadd** [comment]
@Muhayyuddin thank you that is useful information.  To double check, you are able to call from UWP to Chrome M76 (previous version currently) but not from UWP to Chrome M77?

**Comment by Muhayyuddin** [comment]
@jamescadd I have tried with the older version of chrome I think it is the version of December 2018 (Version 71.0.3578.80 (Official Build) (64-bit)) it works perfectly fine. I am able to call from peerCC UWP app to browser also from HoloLens to browser.
I think there is a compatibility issue between webRTC M71 and the webRTC branch in the current version of chrome (that is I think M77).
I will try with UWP PeerCC M75 to browser (latest version) later next week. The issue (that i experienced in the past) was peerCC unity for HoloLense did not work properly with M75 (I was not able to initiate the call from HoloLens to peerCC UWP thats why i switched to M71 unity fix branch). I dont know if this issue is fixed or still there?

**Comment by Muhayyuddin** [comment]
@jamescadd  I have tested with both versions M71 and M75 of PeerCC UWP App. We can successfully communicate from UWP app to the chrome version 71.0.3578.80 (2018-12-11) or older. with the recent version we have the same problem as described above.
